web/modules/custom/dpl_admin/dpl_admin.module

```diff
@@ -228,3 +229,20 @@ function dpl_admin_datetime_set_format(array $element): array {
   $element['time']['#attributes']['step'] = 60;
   return $element;
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * The editor can place links in the menu in several levels, but we only
+ * display the first level in the frontend.
+ * If the editor still places a multi-level link, we'll give them a warning.
+ */
+function dpl_admin_menu_link_content_presave(MenuLinkContent $entity): void {
+  if ('main' !== $entity->getMenuName()) {
+    return;
+  }
+
+  if (!$entity->get('parent')->isEmpty()) {
+    \Drupal::messenger()->addWarning(t('You have added a menu link with a parent. Please notice that this menu link will not be displayed, as we do not support multi-levels in the main menu.'));
```

Comment:
Please remember to add a context to texts. This makes them easier for translators to know where the text is used.